### ISSUE TITLE
FIX Gen3-DRS ENV issue

### ISSUE THREAD

**Issue by Vlad-Dembrovskyi** [TheJacksonLaboratory/splicing-pipelines-nf#290]
## Problem
With Gen3-DRS input option the pipeline is failing at the moment:
https://cloudos.lifebit.ai/public/jobs/6195a5ddfb26cf01dce62a49
https://cloudos.lifebit.ai/public/jobs/619d3eb740695e01e73e9b67

Error message:
```
No such file: ..../.command.env
```
![image](https://user-images.githubusercontent.com/64809705/143300704-2614abaa-d828-48b3-bc06-9079e33b403a.png)
![image](https://user-images.githubusercontent.com/64809705/143300740-2af74902-593d-4446-8e88-43912b9b71ba.png)

Suspicion:
It may be this output `env(sample_name)` that is causing the issue. We had this error before in other pipelines. 
https://github.com/TheJacksonLaboratory/splicing-pipelines-nf/blob/217e202cab3264c9d2d4cafe80b2476a2d837a85/main.nf#L479

## Solution
David suggested change `env(...)` format of output to `stdout` ([example](https://github.com/lifebit-ai/fastq-checks-nf/blob/e4f6d60524e69a2daf54a5d2b467ccb185ae2692/main.nf#L499))
Basically echo the desired value to stdout, and capture stdout.
**Risc:** there may be a lot of stuff printed to stdout.

## Implementation
How will we implement the solution?
- [ ] Step 1
- [ ] Step 2
- [ ] ...
